Backend/dev/test-tool/context-api/server.py

```
@@ -3286,6 +3326,18 @@ def _handle(self, method):
             self._send_json({"started": True, "from": from_env, "forceFetch": force_fetch})
             return True
 
+        if method in ("GET", "POST") and path == "/api/integration-tests/seed-toll-dashboard":
+            self._send_json(seed_toll_dashboard_access())
+            return True
+
+        if method in ("GET", "POST") and path == "/api/local-testing-data/reapply":
+            try:
+                _apply_local_testing_data()
+                self._send_json({"ok": True})
+            except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)